dpd.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "savon", "~> 2.13.0"
   spec.add_dependency "httpclient"
+  spec.add_dependency "rack", "2.2.9"
```

Comment:
Pins to the current version used by the monolith

This resolves an incompatibility issue between Savon and the later version of Rack that was causing tests to fail